src/main/java/meteordevelopment/meteorclient/utils/misc/CPSUtils.java

```diff
@@ -0,0 +1,58 @@
+/*
+ * This file is part of the Meteor Client distribution (https://github.com/MeteorDevelopment/meteor-client).
+ * Copyright (c) Meteor Development.
+ */
+
+package meteordevelopment.meteorclient.utils.misc;
+
+import meteordevelopment.meteorclient.MeteorClient;
+import meteordevelopment.meteorclient.events.world.TickEvent;
+import meteordevelopment.meteorclient.utils.PreInit;
+import meteordevelopment.orbit.EventHandler;
+import meteordevelopment.starscript.value.Value;
+
+
+public class CPSUtils {
+    private static int clicks;
+    private static int cps;
+    private static int secondsClicking;
+    private static long lastTime;
+
+    @PreInit
+    public static void init() {
+        MeteorClient.EVENT_BUS.subscribe(CPSUtils.class);
+    }
+
+    @EventHandler
+    private static void onTick(TickEvent.Pre event) {
+        CPSUtils.cpsChecker();
+    }
+
+
+    public static void onAttack() {
+        CPSUtils.clicks++;
+        CPSUtils.cps++;
+    }
+
+    public static Value getCpsAverage() {
+        return Value.number((int)(
+            clicks / (secondsClicking == 0 ? 1 : (float)secondsClicking)
+        ));
+    }
+
+    public static void cpsChecker() {
```

Comment:
This doesnt need to be a method, and a public at that. You can just move the body to the `onTick` handler.